cassandra-scheduler/src/dist/conf/scheduler.yml

```diff
@@ -199,3 +199,9 @@ logging:
 http_client:
   timeout: ${HTTP_CLIENT_TIMEOUT:-500ms}
   connectionTimeout: ${HTTP_CLIENT_CONNECTION_TIMEOUT:-500ms}
+metrics:
+  host: localhost
+  port: ${METRIC_PORT:-1111}
```

Comment:
Metrics are emitted to StatsD port, which does not need to be part of mesos resource consumption. As there will be one StatsD process running on host, which can take input metrics from multiple processes running on that host.